release-notes/release-notes52.md

```diff
@@ -53,9 +53,11 @@ For information on using the `TextSeeker` API, see: [{{site.productname}} APIs -
 
 ### Updated the colorinput component visual appearance --- updated color swatches in input fields
 
-### Missing resize_img_proportional setting
+### New proportional image resizing setting
```

Comment:
This isn't new and I believe it actually also controls table resizing. I'd double check with @jhaines though.

Reply:
so have it just been fixed? revived?

Reply:
Fixed, as it broke back in I think 4.7.x.

Reply:
release note removed